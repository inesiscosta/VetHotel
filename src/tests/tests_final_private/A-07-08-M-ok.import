√ÅRVORE|T1|Pinheiro 1|5|20|EVERGREEN
HABITAT|h1|nome1|11|T1
HABITAT|h2|nome2|22
