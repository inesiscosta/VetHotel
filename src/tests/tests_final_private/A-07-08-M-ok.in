8
5
h2
T1
werwe
122
4
DECIDUOUS
6
h2
0
0
