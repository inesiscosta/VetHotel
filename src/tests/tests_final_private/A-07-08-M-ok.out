Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Exit
Choose an option: Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Exit
Choose an option: Habitat ID: Tree ID: Tree Name: Tree Age: Tree's Difficulty to Clean: Tree Type: (DECIDUOUS or EVERGREEN) Plant Tree: Invalid Operation: Tree 'T1' already exists.
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Exit
Choose an option: Habitat ID: Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Exit
Choose an option: Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Exit
Choose an option: 